.github/workflows/release.yml

```diff
@@ -43,6 +47,7 @@ jobs:
       container-workdir: /opt/NeMo-Curator
       library-name: NeMo Curator
       dry-run: ${{ inputs.dry-run }}
+      version-bump-branch: ${{ inputs.version-bump-branch }}
```

Comment:
Could you please elaborate on how this works? My guess is that this is for handling automatic updates to the version by creating a new branch(?), but looking into https://github.com/NVIDIA/NeMo-FW-CI-templates/tree/main it's not very clear to me how this works.